dir.props

```diff
@@ -77,8 +77,13 @@
   <!-- Set default Configuration and Platform -->
   <PropertyGroup>
     <Configuration Condition="'$(Configuration)' ==''">Debug</Configuration>
-    <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
-    <PlatformTarget Condition="'$(PlatformTarget)'==''">$(Platform)</PlatformTarget>
+    <Platform Condition="'$(Platform)'==''">x86</Platform>
```

Comment:
That's just the default Platform. Before it was AnyCPU (for everything). Now it's x86, but only on projects that are `PlatformSpecificBuild==true`.